services/spark/helm/spark/values.yaml

```diff
@@ -0,0 +1,83 @@
+master:
+  image:
+    repository: "kubelytics/spark"
+    tag: "2.1.2"
+    pullPolicy: "IfNotPresent"
+    command: /etc/config/spark/run.sh
+  replicas: 1
+  servicePort: 7077
+  containerPort: 7077
+  
+  resources: {}
+    # limits:
+      # cpu: 1
+      # memory: "3Gi"
+    # requests:
+      # cpu: 1
+      # memory: "3Gi"
+  
+  webAdmin:
+    servicePort: 8080
+    containerPort: 8080
+
+worker:
+  image:
+    repository: "kubelytics/spark"
+    tag: "2.1.2"
+    pullPolicy: "IfNotPresent"
+    command: /etc/config/spark/run.sh
+  replicas: 1
+  containerPort: 8081
+  executorMemory: "1g"
+  
+  resources: {}
+    # limits:
+      # cpu: 1
+      # memory: "4Gi"
+    # requests:
+      # cpu: 1
+      # memory: "4Gi"
+
+zeppelin:
+  image:
+    repository: "kubelytics/zeppelin"
+    tag: "0.7.3"
+    pullPolicy: "IfNotPresent"
+    command: /etc/config/zeppelin/run.sh
+  replicas: 1
+  servicePort: 8080
+  containerPort: 8080
+  # nodePort: 32100
+  
+  resources: {}
+    # limits:
+      # cpu: 1
+      # memory: "2Gi"
+    # requests:
+      # cpu: 1
+      # memory: "2Gi"
+
+  preloadNotebooks:
+    enabled: true
```

Comment:
Perhaps we can remove the "enabled" and create notebooks if there are any configured?

Reply:
The flag is used for the `volumeMounts` and `volumes` directives.